astor/code_gen.py

```diff
@@ -584,7 +584,10 @@ def recurse(node):
                         self.write(value.s.replace('{', '{{').replace('}', '}}'))
                     elif isinstance(value, ast.FormattedValue):
                         with self.delimit('{}'):
-                            self.visit(value.value)
+                            if sys.version_info >= (3, 8) and value.expr_text:
```

Comment:
Version checking isn't needed, we can just use `getattr(value, 'expr_text', None)`.

Reply:
Also, please add a comment to explain that we are using `expr_text` to support debugging with f-strings.